quinn-h3/src/connection.rs

```diff
@@ -133,7 +141,10 @@ impl ConnectionInner {
 
         self.reset_waker(cx);
 
-        Ok(self.inner.is_closing() && self.inner.requests_in_flight() == 0)
+        if self.inner.is_closing() && self.inner.requests_in_flight() == 0 {
```

Comment:
Missed this the first time around, but maybe consider hoisting the `Ok()` out and moving the `DriveState::Running` into an `else` branch?

Reply:
This is from the last review addressing work, you couldn't catch it! :)

Reply:
It's better like this, thanks.